packages/UsageAnalysis/files/Test Track/Connections/Import SWAGGER.md

```diff
@@ -1,14 +1,11 @@
-1. Find
-   the [openweathermap.yaml](https://github.com/datagrok-ai/public/blob/master/packages/Samples/swaggers/openweathermap.yaml)
-   file in the Samples package. Press 'Download raw file' to download file to local machine.
-2. Drag-and-drop openweathermap.yaml file from your local storage to the browser tab with opened **Datagrok**.
-3. Go to **Browse > Platform > Functions > Web > OpenWeatherMap**, make sure it is your connection checking tooltip. The tooltip
-   should have Created by your user.
-4. Make sure the OpenWeatherMap connection follows the description in the Swagger File. (TODO: How to check that it
-   follows):
-5. Run all queries for the connection
+1. Find the [fruit-shop.yaml](https://github.com/datagrok-ai/public/blob/master/packages/Samples/swaggers/fruit-shop.yaml) file in the Samples package. 
+2. Press 'Download raw file' to download file to local machine.
+3. Drag-and-drop fruit-shop.yaml file from your local storage to the browser tab with opened **Datagrok**.
+4. Go to **Browse > Platform > Functions > OpenAPI > OpenWeatherMap**, make sure it is your connection checking tooltip. The tooltip should have **Created by* field with your user on it.
```

Comment:
Something is not right :) it says OpenWeatherMap instead of fruit shop

Reply:
changed